File: Class work/3-9-24/task1.c
#include<stdio.h>
main()
{
    int a[5],i,sum=0;

    for(i=0;i<5;i++)
    {
        printf("Enter the number = ");
        scanf("%d",&a[i]);
    }
    for(i=0;i<5;i++)
    {
        sum=sum+a[i];
    }
    printf("sum =%d",sum);

}

File: Class work/Python/16-12-24/task1.py
def myfun():
    n = int(input("Enter the number of Rows: "))

    for i in range(1,n+1):
        print("*"*i)


def myfun2():
    n = int(input("Enter the number of Rows: "))

    for i in range(1,n+1):
        print(" "*(6-i),"*"*i)

def myfun3(n):

    for i in range(1,n+1):
        print(" "*(6-i)," *"*i)

def myfun4(n):
    prime  = 0
    for i in range(1,n+1):

        if n%i == 0:
            
            prime = prime + 1

        if prime==2:
           print(n,"is prime")

        else:
           print(n,"in not prime")
